.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    jest: true,
+  },
+  extends: [
+    'airbnb',
```

Comment:
Extend `prettier` . we don't need airbnb's hostility. make sure you also add a prettier config file.